### PR TITLE
修改COPY-FROM No. 1 others

### PR DESCRIPTION
**Pull request by Ainavo**
将代码块 .. code-block:: 使用 COPY-FROM 指令代替
- https://github.com/PaddlePaddle/Paddle/pull/55411

PADDLEPADDLE_PR=55411

### CODE REVIEW

**Comment by SigureMo** [comment]
可以收到 comment 嘛？

**Comment by Ainavo** [comment]
| API | 对应文档位置 | 备注 |
|--------|--------|--------|
| Tensor.clear_grad | paddlepaddle\Paddle\paddle\fluid\pybind\imperative.cc |  | 
| Tensor.dtype |  |  | 
| Tensor.grad |  |  | 
| Tensor.is_leaf | paddlepaddle\Paddle\paddle\fluid\pybind\imperative.cc |  |
| Tensor.item(*args) | paddlepaddle\Paddle\python\paddle\fluid\dygraph\tensor_patch_methods.py |  |
| Tensor.name |  |  | 
| Tensor.ndim |  |  | 
| Tensor.persistable |  |  | 
| Tensor.place |  |  | 
| Tensor.shape |  API:paddle.shape | | 
| Tensor.stop_gradient |  |  | 
| Tensor.astype(dtype) |  |  | 
| Tensor.backward(grad_tensor=None, retain_graph=False) |  paddlepaddle\Paddle\python\paddle\fluid\dygraph\tensor_patch_methods.py |  |
| Tensor.clear_gradient() | paddlepaddle\Paddle\python\paddle\fluid\framework.py |  | 
| Tensor.clone() | paddlepaddle\Paddle\paddle\fluid\pybind\imperative.cc | |
| Tensor.cosh(name=None) | API:paddle.cosh |  | 
| Tensor.cpu() | paddlepaddle\Paddle\paddle\fluid\pybind\imperative.cc |  |
| Tensor.cuda(device_id=None, blocking=False) | https://github.com/PaddlePaddle/Paddle/blob/4a4a32ee982b4eac6058261a31a79f3a2ae4c845/paddle/fluid/pybind/imperative.cc#L1666 | 示例代码不一致 | 
| Tensor.detach() | https://github.com/PaddlePaddle/Paddle/blob/4a4a32ee982b4eac6058261a31a79f3a2ae4c845/paddle/fluid/pybind/imperative.cc#L1259 | 示例代码不一致 | 
| Tensor.dim() |  |  | 
| Tensor.element_size() | paddlepaddle\Paddle\paddle\fluid\pybind\imperative.cc |  |
| Tensor.exponential_(lam=1.0, name=None) | paddlepaddle\Paddle\python\paddle\tensor\random.py |  |
| Tensor.fill_(x, value, name=None) | paddlepaddle\Paddle\python\paddle\tensor\manipulation.py |  | 
| Tensor.zero_(x, name=None) | paddlepaddle\Paddle\python\paddle\tensor\manipulation.py |  | 
| Tensor.fill_diagonal_(x, value, offset=0, wrap=False, name=None) | https://github.com/PaddlePaddle/Paddle/blob/4a4a32ee982b4eac6058261a31a79f3a2ae4c845/python/paddle/tensor/manipulation.py#L913 | 示例代码不一致 |  
| Tensor.fill_diagonal_tensor(x, y, offset=0, dim1=0, dim2=1, name=None) | paddlepaddle\Paddle\python\paddle\tensor\manipulation.py |  | 
| Tensor.fill_diagonal_tensor_(x, y, offset=0, dim1=0, dim2=1, name=None) | paddlepaddle\Paddle\python\paddle\tensor\manipulation.py |  | 
| Tensor.gradient() |  |  | 
| Tensor.ndimension() |  |  | 
| Tensor.numpy() | paddlepaddle\Paddle\paddle\fluid\pybind\imperative.cc |  | 
| Tensor.pin_memory(y, name=None) | https://github.com/PaddlePaddle/Paddle/blob/4a4a32ee982b4eac6058261a31a79f3a2ae4c845/paddle/fluid/pybind/imperative.cc#L1632 | 示例代码不一致 | 
| Tensor.register_hook(hook) | Paddle\python\paddle\fluid\dygraph\tensor_patch_methods.py |  | 
| Tensor.set_value(value) | paddlepaddle\Paddle\python\paddle\fluid\dygraph\tensor_patch_methods.py |  | 
| Tensor.sinh(name=None) | API:paddle.sinh |  | 
| Tensor.uniform_(min=-1.0, max=1.0, seed=0, name=None) | paddlepaddle\Paddle\python\paddle\tensor\random.py |  |

File: docs/api/paddle/LazyGuard_cn.rst [hunk 1]
```diff
@@ -13,15 +13,4 @@ LazyGuard 是一个用于设置模型（继承自 ``paddle.nn.Layer`` ） 中参
...
-    from paddle.nn import Linear
-
-    with LazyGuard():
-        # w and b are initialized lazily and have no memory.
-        net = Linear(10, 10)
-
-    for param in net.parameters():
-        # Initialize param and allocate memory explicitly.
-        param.initialize()
+COPY-FROM: paddle.LazyGuard_cn
```
Comment (sunzhongkai588) [inline docs/api/paddle/LazyGuard_cn.rst]: [英文源码](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/lazy_init.py#L91)处没有写相应的代码，把这儿的代码先补充到源码吧
image

File: docs/api/paddle/Tensor_cn.rst
Comment (sunzhongkai588) [inline docs/api/paddle/Tensor_cn.rst:1]: tensor_cn.rst 这部分坑有点多..先恢复原状叭，不用改成copy from

File: docs/api/paddle/add_n_cn.rst [hunk 2]
```diff
@@ -10,35 +10,8 @@ add_n
...
-        第二个输入：
-                input2.shape = [2, 3]
-                input2 = [[7, 8, 9],
-                      [10, 11, 12]]
-
-        输出：
-            output.shape = [2, 3]
-            output = [[8, 10, 12],
-                  [14, 16, 18]]
+COPY-FROM: paddle.add_n:code-example1
```
Comment (sunzhongkai588) [inline docs/api/paddle/add_n_cn.rst]: 这部分恢复原状叭，因为是 `.. code-block:: text` 而不是  `.. code-block:: python`，不用改动

File: docs/api/paddle/grad_cn.rst [hunk 3]
```diff
@@ -28,82 +28,6 @@ grad
...
-        # dy1 = [1], dy2 = [4]
-        print(test_dygraph_grad([None, grad_value])) # [16.]
-
-        # dy1 = [4], dy2 = [1]
-        print(test_dygraph_grad([grad_value, None])) # [19.]
-
-        # dy1 = [3], dy2 = [4]
-        grad_y1 = paddle.to_tensor(3.0)
-        print(test_dygraph_grad([grad_y1, grad_value])) # [24.]
+COPY-FROM: paddle.grad
```
Comment (sunzhongkai588) [inline docs/api/paddle/grad_cn.rst]:
```suggestion
COPY-FROM: paddle.grad:code-example-1
COPY-FROM: paddle.grad:code-example-2
```

File: docs/api/paddle/round_cn.rst
Comment (sunzhongkai588) [inline docs/api/paddle/round_cn.rst:1]: 这部分text只是说明，不是代码示例，不用改

File: docs/api/paddle/scatter_cn.rst [hunk 4]
```diff
@@ -7,30 +7,8 @@ scatter
...
-            if (overwrite):
-                x[index[i]] = updates[i]
-            else:
-                x[index[i]] += updates[i]
-        # output:
-        out = paddle.to_tensor([[3, 3], [6, 6], [1, 1]])
-        out.shape # [3, 2]
-
-**Notice：**
+COPY-FROM: paddle.scatter:code-example1
```
Comment (sunzhongkai588) [inline docs/api/paddle/scatter_cn.rst:10]: 应该在[源码](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/tensor/manipulation.py#L2848)处的两个代码块那儿，分别加上`name`标识符，这也才可以区分不同代码片段，然后copy from的时候可以引用指定的代码片段。
image

File: docs/api/paddle/squeeze_cn.rst
Comment (sunzhongkai588) [inline docs/api/paddle/squeeze_cn.rst:1]: 这部分是text，不用改动

File: docs/api/paddle/stack_cn.rst
Comment (sunzhongkai588) [inline docs/api/paddle/stack_cn.rst:1]: 这部分是text，不用改动

**Comment by Ainavo** [comment]
@sunzhongkai588 辛苦孙师傅有空 review 一下

**Review by sunzhongkai588 — APPROVED** [review]
LGTM
@SigureMo @megemini Tensor_cn.rst下的代码示例就不改成copy from形式了，因为改动难度比较大（示例存在于c++端）。建议直接skip叭
